addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1174,14 +1147,14 @@ var SnippetsMenu = Widget.extend({
             ev.preventDefault();
             ev.stopPropagation();
             ev.stopImmediatePropagation();
+            const $target = $(ev.target);
+            const $defaultSnippetText = $target.closest('.o_default_snippet_text');
             const autoSelectDefaultText = async (context) => {
-                const $target = $(ev.target);
-                const $defaultSnippetText = $target.closest('.o_default_snippet_text');
-                await this.editorHelpers.removeClass(context, $defaultSnippetText[0], 'o_default_snippet_text');
                 const nodes = this.editorHelpers.getNodes($target[0]);
                 this.wysiwyg.editor.selection.select(nodes[0], nodes[nodes.length - 1]);
+                this.wysiwyg.withDomMutations($defaultSnippetText, () => $defaultSnippetText.removeClass('o_default_snippet_text'))
             };
-            this.wysiwyg.editor.execCommand(autoSelectDefaultText);
+            this.wysiwyg.execCommand(autoSelectDefaultText);
```

Comment:
Si c'est nous qui avons mis cette fonction dans un `const`, je préférerais qu'on la remette inline comme avant qu'on y touche.

Reply:
The const is there for seeing the name in the devtool. Do you still want to inline it?

Reply:
Right, right, keep it there then.